regexes/device/mobiles.yml

```diff
@@ -8666,6 +8693,12 @@ Symphony:
 
 # Qtek
 Qtek:
+  regex: '(Q09) Build'
+  device: 'smartphone'
+  model: '$1'
+
+# Q-Touch
+Q-Touch:
```

Comment:
guess that's at an incorrect position. the new regex for `Q09` should be below `Q-Touch`. You moved the `Qtek` detections to the new one...